test/performance/README.md

```diff
@@ -1,128 +1,189 @@
-    run_ID="1" # run ID for your experiment 
-    freq="1"   # workload frequency parameter set in your experiment 
-    numPods=20 # total number of workload objects set in your experiment
-    method = "exp-RandomizedLoad" # clusterloader method used to generate your traffic
-    watch_interval = 3600  # watch for events that to occur within time interval threshold
-```
+   ```bash
+      kubeconfig = "kscore-config"  # path to the kubeconfig file
+      ns = "stress-test"  # name of the namespace where the workload are generated 
+      mypath="data/"  # path to the directory for the output files
+      run_ID="1" # run ID for your experiment 
+      freq="1"   # workload frequency parameter set in your experiment 
+      numPods=20 # total number of workload objects set in your experiment
+      method = "exp-RandomizedLoad" # clusterloader method used to generate your traffic
+      watch_interval = 3600  # watch for events that to occur within time interval threshold
+   ```
 
 3. Run the metrics collection script:
 
-a) Clone the KubeStellar repo:
+   a) Clone the KubeStellar repo:
 
-```bash
-$ git clone https://github.com/kubestellar/kubestellar.git
-$ cd kubestellar/test/performance
-```
+   ```bash
+   $ git clone https://github.com/kubestellar/kubestellar.git
+   $ cd kubestellar/test/performance
+   ```
 
-b) Start collecting metrics:
+   b) Start collecting metrics:
 
-```bash 
-$ python3 metrics_collector.py
-```
+   ```bash 
+   $ python3 metrics_collector.py
+   ```
 
-Metrics will be collected for pods in `wds1`, manifestworks & workstatuses in `its1` and pods & appliedmanifestworks in `wec1` - In total 5 files will be output, for example:
+   Metrics will be collected for pods in `wds1`, manifestworks & workstatuses in `its1` and pods & appliedmanifestworks in `wec1` - In total 5 files will be output, for example:
 
-```bash 
-      20 exp-RandomizedLoad-freq-20-appliedmanifestwork-wec1-run1.txt
-      20 exp-RandomizedLoad-freq-20-manifestwork-its1-run1.txt
-      20 exp-RandomizedLoad-freq-20-pod-wds-kscore-stage-run1.txt
-      20 exp-RandomizedLoad-freq-20-pod-wec1-run1.txt
-      20 exp-RandomizedLoad-freq-20-workstatus-its1-run1.txt
-```
+   ```bash 
+         20 exp-RandomizedLoad-freq-20-appliedmanifestwork-wec1-run1.txt
+         20 exp-RandomizedLoad-freq-20-manifestwork-its1-run1.txt
+         20 exp-RandomizedLoad-freq-20-pod-wds-kscore-stage-run1.txt
+         20 exp-RandomizedLoad-freq-20-pod-wec1-run1.txt
+         20 exp-RandomizedLoad-freq-20-workstatus-its1-run1.txt
+   ```
 
-The output is a file with the following structure:
+   The output is a file with the following structure:
 
-```bash
-<obj_name> <obj_creation_time> <obj_status_update_time> <obj_status_condition>  <obj_status_condition>  <obj_controller_manager>
-```
+   ```bash
+   <obj_name> <obj_creation_time> <obj_status_update_time> <obj_status_condition>  <obj_status_condition>  <obj_controller_manager>
```

Comment:
Why is `<obj_status_condition>` listed twice?

Reply:
Removing the context of that subsection for now as it will be completely revised in a NEW PR, as already stated on my comment above. Sorry for the confusion!